### PR TITLE
Feature/add prerelease flag

### PR DESCRIPTION
**Pull request by jonnyreeves**
This PR contains two commits, the first, 8bdab49, is a refactor of the existing code to support an arbitrary number of flags and the second, efc7eef, introduces a new `-prelease` flag.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -100,13 +103,14 @@ func main() {
 		return
 	}
 
-	if len(os.Args) < 6 {
+	if flag.NArg() != 5 {
+		log.Printf("Error: Invalid number of arguments (got %d, expected 5)\n\n", flag.NArg())
```
Comment (jonnyreeves) [inline main.go:107]: This one caught me out a few times when first on-boarding with this command (as no specific error was printed).

**Comment by jonnyreeves** [comment]
@c4milo any chance of a release please?

**Comment by c4milo** [comment]
@jonnyreeves yes! I'll schedule it for tonight. Apologies for the delay.

**Comment by c4milo** [comment]
@jonnyreeves it landed in https://github.com/c4milo/github-release/releases/tag/v1.1.0 Thanks again for your contribution!

**Pull request merged**